### ISSUE TITLE
beta.11 regression: I18N interface no longer has an `install`function

### ISSUE THREAD

**Issue by cexbrayat** [intlify/vue-i18n-next#225]
### vue & vue-i18n version
ex:
- vue: 3.0.4
- vue-i18n: 9.0.0-beta.11

### Reproduction Link
https://github.com/cexbrayat/vue-cli-playground/pull/2

### Steps to reproduce

`yarn` and `yarn serve` throws:
```ts
TS2345: Argument of type 'I18n<{ en: { hello: string; }; }, unknown, unknown, true>' is not assignable to parameter of type 'Plugin_2'.
  Property 'install' is missing in type 'I18n<{ en: { hello: string; }; }, unknown, unknown, true>' but required in type '{ install: PluginInstallFunction; }'.
    4 |
    5 | createApp(App)
  > 6 |   .use(i18n)
      |        ^^^^
    7 |   .mount('#app')
```

Unit tests also fail.

It appears that the `I18n`  interface used to have an `install` property, but no longer has one in beta.11.

### What is Expected?

We should be able to use the i18n plugin with TS

### What is actually happening?

We can't use the plugin as it does not compile with beta.11

**Comment by Alymbek** [comment]
I just updateed to beta-11 and getting below error message:

```
Module build failed (from ./node_modules/@intlify/vue-i18n-loader/lib/index.js):
TypeError: vue_i18n_1.baseCompile is not a function
```

**Comment by npm-ued** [comment]
@cexbrayat me too

**Comment by npm-ued** [comment]
vue-i18n 9.0.0-beta.11
vue 3.03

**Comment by kazupon** [comment]
> I just updateed to beta-11 and getting below error message:
> 
> ```
> Module build failed (from ./node_modules/@intlify/vue-i18n-loader/lib/index.js):
> TypeError: vue_i18n_1.baseCompile is not a function
> ```

As mention at #222, In beta.11, I released the re-packed modules with monorepo to solve the CSP problem.

I'm working on bundler loader /plugins.

**Comment by hezechang** [comment]
me too

Argument of type 'I18n<{}, {}, {}, true>' is not assignable to parameter of type 'Plugin_2'.
  Property 'install' is missing in type 'I18n<{}, {}, {}, true>' but required in type '{ install: PluginInstallFunction; }'

**Comment by kayahasa** [comment]
me too 
src/main.ts:50:9
TS2345: Argument of type 'I18n<{ en: { darkmode: string; }; ja: { darkmode: string; }; }, unknown, unknown, true>' is not assignable to parameter of type 'Plugin_2'.
  Property 'install' is missing in type 'I18n<{ en: { darkmode: string; }; ja: { darkmode: string; }; }, unknown, unknown, true>' but required in type '{ install: PluginInstallFunction; }'.

**Comment by kazupon** [comment]
I've just released out 9.0.0-beta.12
fixed `i18n` interface `install` type missing.
please check it!

**Comment by meddario** [comment]
Hi, 
this seems still relevant with version `9.0.0-beta.16`:

```
Argument of type '{ i18n: I18n<{ en: { message: { hello: string; }; }; it: { message: { hello: string; }; }; }, unknown, unknown, true>; }' is not assignable to parameter of type 'Plugin_2'.
  Property 'install' is missing in type '{ i18n: I18n<{ en: { message: { hello: string; }; }; it: { message: { hello: string; }; }; }, unknown, unknown, true>; }' but required in type '{ install: PluginInstallFunction; }'.ts(2345)
runtime-core.d.ts(889, 5): 'install' is declared here.
```

Is there any workaround?
